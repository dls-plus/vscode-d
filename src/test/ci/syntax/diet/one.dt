tag.class#id(attr=code, attr=4, attr="value") hello world