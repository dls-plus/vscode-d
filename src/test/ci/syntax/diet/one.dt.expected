[{"start":0,"end":3,"scope":"entity.name.tag.diet"},{"start":3,"end":9,"scope":"constant.other.class.diet"},{"start":9,"end":12,"scope":"constant.other.id.diet"},{"start":12,"end":13,"scope":"constant.name.attribute.tag.diet"},{"start":13,"end":17,"scope":"entity.other.attribute-name.tag.diet"},{"start":17,"end":18,"scope":"attribute_value"},{"start":18,"end":22,"scope":"variable.d"},{"start":22,"end":24,"scope":"meta.tag.other"},{"start":24,"end":28,"scope":"entity.other.attribute-name.tag.diet"},{"start":28,"end":29,"scope":"attribute_value"},{"start":29,"end":30,"scope":"constant.numeric.integer.decimal.d"},{"start":30,"end":32,"scope":"meta.tag.other"},{"start":32,"end":36,"scope":"entity.other.attribute-name.tag.diet"},{"start":36,"end":37,"scope":"attribute_value"},{"start":37,"end":38,"scope":"string.quoted.diet"},{"start":38,"end":43,"scope":"string.quoted.diet"},{"start":43,"end":44,"scope":"string.quoted.diet"},{"start":44,"end":45,"scope":"constant.name.attribute.tag.diet"},{"start":45,"end":57,"scope":"source.diet"}]